### PR TITLE
[IIO] Handling queries for tensor src IIO

### PR DESCRIPTION
**Pull request by kparichay**
Handling scheduling query and remaining queries handled with basesrc class
All events are handled by basesrc class

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

File: gst/nnstreamer/tensor_source/tensor_src_iio.c [hunk 1]
```diff
@@ -428,7 +428,7 @@ gst_tensor_src_iio_stop (GstBaseSrc * src)
 static gboolean
 gst_tensor_src_iio_event (GstBaseSrc * src, GstEvent * event)
 {
-  //FIXME: fill this function
+  /* No events to be handled yet */
```
Comment (leemgs) [inline gst/nnstreamer/tensor_source/tensor_src_iio.c]: Please use a **/**  .... */** style to generate doxygen document correctly ASAP.

**Comment by wooksong** [comment]
You may need another commit for replacing all of the '//FIXME: fill this function' lines with  '/** FIXME: fill this function */'. Anyway this PR looks good to me it self.

File: gst/nnstreamer/tensor_source/tensor_src_iio.c (hunk 1 above)
Comment (kparichay) [reply gst/nnstreamer/tensor_source/tensor_src_iio.c]: Yes, on it.

**Comment by kparichay** [comment]
> You may need another commit for replacing all of the '//FIXME: fill this function' lines with '/** FIXME: fill this function */'. Anyway this PR looks good to me it self.

Updated with commit https://github.com/nnsuite/nnstreamer/pull/1210/commits/df81d930611bb88d9d1e4ceb77d7bdbe42c3a206

**Comment by myungjoo** [comment]
@kparichay Please resolve the merge conflict.
```
$ git checkout master
$ git pull $GITHUBMAIN master
$ git checkout tensor_src_iio_query_event
$ git rebase master
$ .... resolve conf ... (e.g., git mergetool if you have configured mergetool for git)
$ git rebase --continue
$ git push $GITHUBFORK HEAD -f
```

**Comment by kparichay** [comment]
re-based and re-synced

**Pull request merged**